pkg/terraform/state/deploy.go

```diff
@@ -241,9 +244,9 @@ func (h *handler) createJob(or []metaV1.OwnerReference, input *Input, runName, r
 
 func (h *handler) createServiceAccount(or []metaV1.OwnerReference, name, namespace string) (*coreV1.ServiceAccount, error) {
 	meta := metaV1.ObjectMeta{
-		Name:            "sa-" + name,
-		Namespace:       namespace,
-		OwnerReferences: or,
+		Name:      "sa-" + name,
+		Namespace: namespace,
+		//OwnerReferences: or,
```

Comment:
This will cause orphans if something else fails along the line before getting to `updateOwnerReference`

Reply:
removing for now, seems like a bug in k8s or something. @Oats87 figured most of this out in this convo https://kubernetes.slack.com/archives/C0EG7JC6T/p1562794268483900